addons/website_livechat/static/src/web/livechat_channel_info_list_patch.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website_livechat.LivechatChannelInfoList" t-inherit="im_livechat.LivechatChannelInfoList" t-inherit-mode="extension">
+        <xpath expr="//t[@t-name='extra_infos']" position="inside">
+            <div t-if="recentConversations.length" class="d-flex flex-column bg-inherit gap-1">
+                <h3 class="pt-3">Recent conversations</h3>
+                <t t-foreach="recentConversations" t-as="thread" t-key="thread.localId">
+                    <button
+                        class="btn btn-sm btn-secondary d-flex align-items-center justify-content-center gap-1 rounded-3"
+                        t-on-click="()=>thread.open()"
+                    >
+                        <i class="fa fa-external-link opacity-75"/>
+                        <span class="ms-1 fw-bold" t-esc="thread.displayName"/>
```

Comment:
displayName of conversation is not that useful: it's redundant with current conversation name. At the very least putting the create_date is a better way to see the different conversations.

I think it'd nice to have a similar look than sessions in the backend: show data, duration, amount of messages, quick status and avatar stack of people:
<img width="893" alt="Screenshot 2025-06-30 at 17 21 02" src="https://github.com/user-attachments/assets/60855b30-094b-4612-865f-0f846cab8678" />

But we can plan better looking item in follow-up PR.